apps/constellation/config_builder.cpp

```diff
@@ -68,11 +68,14 @@ Constellation::Config BuildConstellationConfig(Settings const &settings)
   cfg.max_peers             = settings.max_peers.value();
   cfg.transient_peers       = settings.transient_peers.value();
   cfg.block_interval_ms     = settings.block_interval.value();
+  cfg.aeon_period           = settings.aeon_period.value();
+  cfg.max_committee_size    = settings.max_committee_size.value();
+  cfg.stake_delay_period    = settings.stake_delay_period.value();
   cfg.peers_update_cycle_ms = settings.peer_update_interval.value();
   cfg.disable_signing       = settings.disable_signing.value();
   cfg.sign_broadcasts       = false;
-  cfg.load_state_file       = settings.load_state.value();
-  cfg.stakefile_location    = settings.stakefile_location.value();
+  cfg.load_genesis_file     = settings.load_genesis_file.value();
+  cfg.genesis_file_location = settings.genesis_file_location.value();
   cfg.proof_of_stake        = settings.proof_of_stake.value();
   cfg.network_mode          = GetNetworkMode(settings);
   cfg.features              = settings.experimental_features.value();
```

Comment:
I think we need to work out which of these configuration settings belongs in the genesis file. It is probably worth creating an issue for this and marking it as tbd before mainnet.

Reply:
Noted